examples/jsm/loaders/3DMLoader.js

```diff
@@ -226,7 +226,8 @@ class Rhino3dmLoader extends Loader {
 
 			return new MeshStandardMaterial( {
 				color: new Color( 1, 1, 1 ),
-				metalness: 0.8,
+				metalness: 0.1,
+				roughness: 0.6,
```

Comment:
> Also modify the default MeshStandardMaterial params (this is arbitrary)

Since they are arbitrary, please revert these changes.

Reply:
It has been reverted.